master/usage/assigning-ip-addresses-topology.md

```diff
@@ -0,0 +1,135 @@
+This may be desirable either to reduce the number of routes required in the network or to meet requirements imposed by an external firewall device or policy.
+This guide walkthroughs an example of how to set this up.
+
+## Prerequisites
+
+**{{site.prodname}} IPAM**
+
+This guide only applies if you are using {{site.prodname}} IPAM.
+
+**Labeled Nodes**
+
+In order to assign IP pools to specific Nodes, these Nodes must be labeled. See the documentation for [calicoctl label]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/label) and [kubectl label](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes/#add-a-label-to-a-node) for more information on how to do this. We recommend using `kubectl label`.
+
+### Example: Kubernetes
+
+In this example, we created a cluster with 4 nodes across 2 racks (2 nodes/rack). Consider the following:
+
+```
+       -------------------
+       |    router       |
+       -------------------
+       |                 |
+---------------   ---------------
+| rack-0      |   | rack-1      |
+---------------   ---------------
+| kube-node-0 |   | kube-node-2 |
+- - - - - - - -   - - - - - - - -
+| kube-node-1 |   | kube-node-3 |
+- - - - - - - -   - - - - - - - -
+```
+
+Using the pods IP range `192.168.0.0/16`, we target the following setup: reservet the `192.168.0.0/24` and `192.168.1.0/24` blocks for `rack-0`, `rack-1`, `rack-2`, and `rack-3` respectively.
+Let's get started.
+
+
+By installing {{ site.prodname }} without setting the
+default IP pool to match, running `calicoctl get ippool -o wide` shows that {{site.prodname}} 
+created its default IP pool of `192.168.0.0/16`:
+
+```
+NAME                  CIDR             NAT    IPIPMODE   DISABLED
+default-ipv4-ippool   192.168.0.0/16   true   Always     false
+```
+{: .no-select-button}
+
+1. Delete the default IP pool.
+
+	Since the `default-ipv4-ippool` IP pool resource already exists and accounts for the entire `/16` block, we will have to delete this first:
+
+   ```
+   calicoctl delete ippools default-ipv4-ippool
+   ```
+
+2. Label the nodes.
+   ```
+   kubectl label nodes kube-node-0 rack=0
+   kubectl label nodes kube-node-1 rack=0
+   kubectl label nodes kube-node-2 rack=1
+   kubectl label nodes kube-node-3 rack=1 
+   ```
+3. Create an IP pool for each rack.
+
+   ```
+   calicoctl create -f -<<EOF
+   apiVersion: projectcalico.org/v3
+   kind: IPPool
+   metadata:
+     name: rack-0-ippool
+   spec:
+     cidr: 192.168.0.0/24
+     ipipMode: Always
+     natOutgoing: true
+     nodeSelector: rack == "0"
+EOF
+   ```
+
+   ```
+   calicoctl create -f -<<EOF
+   apiVersion: projectcalico.org/v3
+   kind: IPPool
+   metadata:
+     name: rack-1-ippool
+   spec:
+     cidr: 192.168.1.0/24
+     ipipMode: Always
+     natOutgoing: true
+     nodeSelector: rack == "1"
+EOF
+   ```
+
+   We should now have 2 enabled IP pools, which we can see when running `calicoctl get ippool -o wide`:
```

Comment:
As noted in the style guide, if the number is below 10, we go ahead and spell it out so 
s/2/two